backend/v3/storage/database/repository/repository.go

```diff
@@ -5,3 +5,14 @@ import "github.com/zitadel/zitadel/backend/v3/storage/database"
 type repository struct {
 	client database.QueryExecutor
 }
+
+func (r repository) writeCondition(
```

Comment:
providing the functionality as function is enough, a method is an overkill here.
```suggestion
func writeCondition(
```